src/components/header.js

```diff
@@ -111,10 +143,7 @@ function Header({ slug }) {
         id="navDropDownTwo"
       />
     </React.Fragment>,
-    <Link variant="nav" to="/three" key="3">
-      <span>{t("header.nav.parentLinks.0")}</span>
-    </Link>,
-  ];
+  ].concat(parentLinkItems);
```

Comment:
How is the concat functioning here / what is it's purpose

Reply:
The purpose is to add the parent links to the navigation bar. The parent links are stored in their own array to account for when the user wants to have more than 1 parent link.